test/cli/test_cli_flow.sh

```diff
@@ -26,6 +26,7 @@ function post_install_tests {
     check_pgdb_config_override
     test_noobaa_cr_deletion
     test_noobaa_loadbalancer_source_subnet
+    check_default_backingstore
```

Comment:
Can you add a comment that specifies the changes this test does - delete bs, delete firs.bucket, etc.

Reply:
Done. Thanks!